yarn-project/end-to-end/src/e2e_l1_to_l2_msg.test.ts

```diff
@@ -102,15 +106,9 @@ describe('e2e_l1_to_l2_msg', () => {
     return contract;
   };
 
-  it('Milestone 2.2: L1->L2 Calls', async () => {
-    const initialBalance = 10n;
-    const [ownerAddress, receiver] = accounts;
-    const ownerPub = await aztecRpcServer.getAccountPublicKey(ownerAddress);
-    const deployedL2Contract = await deployContract(initialBalance, pointToPublicKey(ownerPub));
-    await expectBalance(accounts[0], initialBalance);
-
-    const l2TokenAddress = deployedL2Contract.address.toString() as `0x${string}`;
+  const delay = (ms: number) => new Promise(resolve => setTimeout(resolve, ms));
 
+  const sendL1ToL2MessageViaPortal = async (ownerAddress: AztecAddress, l2TokenAddress: `0x${string}`) => {
```

Comment:
With the function also including an initialise and token minting etc, the name might need to be extended a bit as it is not conveyed that this is happening.